### PR TITLE
Search component

### PR DESCRIPTION
**Pull request by joseleoc**
A search component with them and i18n

### CODE REVIEW

File: components/molecules/search/index.tsx
Comment (Raidenmystery) [inline components/molecules/search/index.tsx:1]: @joseleoc Create a folder with the name of the component, add two files (index.tsx and Serach.tsx)

    in the index only export the component without any more code or logic
    in the Serach.tsx the rest of the content of the search file

Comment (joseleoc) [reply components/molecules/search/index.tsx:1]: Done

Comment (joseleoc) [reply components/molecules/search/index.tsx:1]: image

Comment (joseleoc) [reply components/molecules/search/index.tsx:1]: image
image

**Review by Raidenmystery — APPROVED** [review]

**Comment by lagoonStudios** [comment]
This PR is clossed due to rebase conflicts, created a new PR https://github.com/lagoonStudios/colavora/pull/13.

**Pull request closed without merging**